src/chronos/utils/date-compare.ts

```diff
@@ -33,6 +33,11 @@ export function isBefore(
   return endOf(date1, units).valueOf() < date2.valueOf();
 }
 
+export function isDisabledDay(date: Date, daysDisabled: number[]):boolean{
+  if(daysDisabled === undefined) return false;
+  return (daysDisabled.filter(d => d == date.getDay()).length > 0);
```

Comment:
Replace with `.any()` we don't actually need an array of results, we just need to know that there are at least one match.
Plus if disabled days array is empty we can `return false;` too